### PR TITLE
Import bus connections

### PR DESCRIPTION
**Pull request by adeas31**
### Purpose

Load the bus connections when importing the SSP.

### Type of Change

- Code refactoring (non-breaking change which improves maintainability)
- Bug fix (non-breaking change which fixes an issue)

### Checklist

- I have performed a self-review of my own code
- I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/OMSimulatorLib/System.cpp [hunk 1]
```diff
@@ -1168,3 +1155,47 @@ oms_status_enu_t oms3::System::updateDependencyGraphs()
 
   return oms_status_ok;
 }
+
+oms_status_enu_t oms3::System::importFromSSD_ConnectionGeometry(const pugi::xml_node& node, const ComRef& crefA, const ComRef& crefB)
```
Comment (lochel) [inline src/OMSimulatorLib/System.cpp]: Should this be better moved to `oms2::ssd:ConnectionGeometry`? And it should probably be copied to `oms3::ssd:ConnectionGeometry`.

Comment (adeas31) [reply src/OMSimulatorLib/System.cpp]: I added `oms2::ssd::ConnectionGeometry::importFromSSD` but I think we still need this function. Most of the functionality is moved to its respective class.

**Comment by OpenModelica-Hudson** [comment]
The following pull requests are now being [tested](https://test.openmodelica.org/hudson/job/OMSimulator-PR/760//console). It may take a while, depending on the workload.
  - OpenModelica/OMSimulator#440
  - OpenModelica/OMSimulator-testsuite#103

The changes will be pushed if all tests run successfully.

**Comment by OpenModelica-Hudson** [comment]
All tests were successful. :confetti_ball:

**Pull request closed without merging**